WordPress/src/main/java/org/wordpress/android/ui/prefs/DotComSiteSettings.java

```diff
@@ -216,12 +349,125 @@ public void onResponse(JSONObject response) {
+                        mRemoteJpSettings.jetpackProtectEnabled = response.optBoolean("active");
+                        mJpSettings.jetpackProtectEnabled = mRemoteJpSettings.jetpackProtectEnabled;
+                        String status = mJpSettings.jetpackProtectEnabled ? "activated" : "deactivated";
+                        AppLog.d(AppLog.T.API, "Jetpack Protect module " + status);
+                        notifySavedOnUiThread(null);
+                    }
+                }, new RestRequest.ErrorListener() {
+                    @Override
+                    public void onErrorResponse(VolleyError error) {
+                        AppLog.w(AppLog.T.API, "Error updating Jetpack Protect module: " + error);
+                    }
+                });
+    }
+
+    private void pushJetpackMonitorSettings() {
+        WordPress.getRestClientUtils().setJetpackMonitor(
+                mSite.getSiteId(), mJpSettings.monitorActive, new RestRequest.Listener() {
+                    @Override
+                    public void onResponse(JSONObject response) {
+                        mRemoteJpSettings.monitorActive = response.optBoolean("active");
+                        mJpSettings.monitorActive = mRemoteJpSettings.monitorActive;
+                        String status = mJpSettings.jetpackProtectEnabled ? "activated" : "deactivated";
+                        AppLog.d(AppLog.T.API, "Jetpack Monitor module " + status);
+                        notifySavedOnUiThread(null);
+                    }
+                }, new RestRequest.ErrorListener() {
+                    @Override
+                    public void onErrorResponse(VolleyError error) {
+                        AppLog.w(AppLog.T.API, "Error updating Jetpack Monitor module: " + error);
+                        notifySavedOnUiThread(error);
+                    }
+                });
+
+        final Map<String, String> params = serializeJetpackParams();
+        if (params == null || params.isEmpty()) return;
+
+        WordPress.getRestClientUtils().setJetpackSettings(
+                mSite.getSiteId(), new RestRequest.Listener() {
+                    @Override
+                    public void onResponse(JSONObject response) {
+                        AppLog.d(AppLog.T.API, "Jetpack Monitor module options updated");
+                        mRemoteJpSettings.emailNotifications = mJpSettings.emailNotifications;
+                        mRemoteJpSettings.wpNotifications = mJpSettings.wpNotifications;
+                        notifySavedOnUiThread(null);
+                    }
+                }, new RestRequest.ErrorListener() {
+                    @Override
+                    public void onErrorResponse(VolleyError error) {
+                        AppLog.w(AppLog.T.API, "Error updating Jetpack Monitor module options: " + error);
+                        notifySavedOnUiThread(error);
+                    }
+                }, params);
```

Comment:
I thought we fixed this in another PR, but maybe it was a different method. We should always have the callbacks as the last parameters for ease of read. Can we move the params to after site id parameter?